Containerized Docker Application Lifecycle - 06.1 (#10621)
dotnet/docs: docs/standard/containerized-lifecycle-architecture/run-manage-monitor-docker-environments/index.md

This last pillar in the containerized Docker applications life cycle is centered on how you can run, manage, and monitor your applications in scalable, high availability (HA) production environments.

> The way you run your containerized applications in production (infrastructure architecture and platform technologies) is  very much related and founded on the chosen architecture and development platforms we looked at in Chapter 1 of this e-book.

Reviewer: ```suggestion
The way you run your containerized applications in production (infrastructure architecture and platform technologies) is very much related and based on the chosen architecture and development platforms discussed in Chapter 1 of this e-book.
```

Author: Yep, sorry, sometimes spanglish creeps in 😅
